starter-code/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "starter-code",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "dev": "nodemon ./server.js -e .js,.hbs",
```

Comment:
mjs.com/package/nodemon/v/1.19.2#specifying-extension-watch-list
`nodemon ./server.js -e .js,.hbs`
Should be 
`nodemon ./server.js -e js,hbs`
According to the docs, might still work though